src/ui/default-prop-types.js

```diff
@@ -173,3 +177,44 @@ export const formItemLayoutShape = PropTypes.shape({
     sm: formItemDimensionShape.isRequired
   }).isRequired
 });
+
+export const importTaskShape = PropTypes.shape({
+  _id: PropTypes.string.isRequired,
+  batchId: PropTypes.string.isRequired,
+  taskType: PropTypes.oneOf([TASK_TYPE.importDocument]),
+  processed: true,
+  attempts: [
+    {
+      startedOn: PropTypes.string,
+      completedOn: PropTypes.string,
+      errors: PropTypes.arrayOf(PropTypes.string).isRequired
+    }
+  ],
+  taskParams: {
+    key: PropTypes.string.isRequired,
+    title: PropTypes.string.isRequired,
+    slug: PropTypes.string.isRequired,
+    language: PropTypes.string.isRequired,
+    updatedOn: PropTypes.string,
+    importedRevision: PropTypes.string,
+    importableRevision: PropTypes.string.isRequired,
+    importType: PropTypes.oneOf([DOCUMENT_IMPORT_TYPE.add, DOCUMENT_IMPORT_TYPE.update])
+  }
+});
+
+export const importBatchShape = PropTypes.shape({
+  _id: PropTypes.string.isRequired,
+  createdBy: userInDocShape.isRequired,
+  createdOn: PropTypes.string.isRequired,
+  completedOn: PropTypes.string,
+  batchType: PropTypes.oneOf([BATCH_TYPE.importDocuments]),
```

Comment:
And same remark like here https://github.com/educandu/educandu/pull/55/files#r759999704 with this guy too 😊

Reply:
Resolved all